src/instructions/Instructions.test.js

```diff
@@ -0,0 +1,23 @@
+import { render, fireEvent } from "@testing-library/react";
+import Toggle from "./Toggle";
+import { Instructions } from "./Instructions";
+
+test("Button displays correct content only when toggled", () => {
+  const { getByText, queryByText } = render(
+    <Toggle>
+      <Instructions />
+    </Toggle>
+  );
+
+  const button = getByText("Instructions");
+
+  expect(queryByText(/Here are the instructions to the game!/)).toBeNull();
+
+  fireEvent.click(button);
+  expect(
+    queryByText(/Here are the instructions to the game!/)
+  ).toBeInTheDocument();
+
+  fireEvent.click(button);
+  expect(queryByText(/Here are the instructions to the game!/)).toBeNull();
+});
```

Comment:
This is a nice intro to writing a test that tests conditionals / logic / behavioral aspects which is the toggling of instructions. 
I'm thinking you could also test that when you click it again, it doesn't show the toggle. So you are testing both ON and OFF toggling functionality.

Reply:
I have both ON and OFF being tested but they are in one test, so I will refactor this into two separate tests. I think this is a perfect opportunity to create my first describe block! 🥳

Reply:
Oops sorry then i missed that. Yes that could make it more explicit.